project/Dependencies.scala

```diff
@@ -114,7 +114,7 @@ object Dependencies {
 
   val slick: ModuleID =           "com.typesafe.slick"  %% "slick"                % slickV excludeAll (excludeTypesafeConfig, excludeReactiveStream)
   val hikariCP: ModuleID =        "com.typesafe.slick"  %% "slick-hikaricp"       % slickV excludeAll (excludeSlf4j)
-  val mysql: ModuleID =           "mysql"               % "mysql-connector-java"  % "8.0.22"
+  val mysql: ModuleID =           "mysql"               % "mysql-connector-java"  % "8.0.28"
```

Comment:
This might cause issue with a database that has existing data. Validating that is a bit of work. Maybe we can just try it out, but keep an eye on dev deploy after this PR merge. If there's issue, we can easily revert dev and make a new PR for reverting the upgrade afterwards

Reply:
Ah good to know. What kind of issues with existing DB have you experienced due to a minor upgrade like this?
Sounds good to merge and monitor.